src/components/Button/Button.module.scss

```
@@ -0,0 +1,34 @@
+@import "@styles/variables.scss";
+@import "@styles/mixins.scss";
+
+.button {
+    display: flex;
+    align-items: center;
+    box-sizing: border-box;
+    border: none;
+    cursor: pointer;
+}
+
+.button_color-primary {
+    width: 108px;
+    height: 41px;
+    background: $color-primary;
+    border-radius: $br;
+    color: $color-light;
+    justify-content: center;
+}
+
+.button_color-secondary {
+    width: 176px;
+    height: 36px;
+    background: $color-gradient;
+    border-radius: $br;
+    color: $color-dark;
+    padding: 0 20px;
+    justify-content: space-between;
+}
+
+.button_disabled {
```

используем scss-приемущества
```suggestion
.button {
    ....
    &_color-primary {
    width: 108px;
    height: 41px;
    background: $color-primary;
    border-radius: $br;
    color: $color-light;
    justify-content: center;
    }
}
```